src/Standards/Generic/Docs/Functions/OpeningFunctionBraceKernighanRitchieStandard.xml

```diff
@@ -1,22 +1,21 @@
-<documentation title="Opening Brace in Function Declarations">
+<documentation title="Opening Function Brace Kerninghan Ritchie">
     <standard>
     <![CDATA[
-    Function declarations follow the "Kernighan/Ritchie style". The function brace is on the same line as the function declaration. One space is required between the closing parenthesis and the brace.
+    The function opening brace must be on the same line as the function declaration closing
+    parenthesis, with exactly one space between them, and must be the last content on the line.
```

Comment:
I think this makes things more confusing for people. I understand the reason why you changed it (multi-line declarations), but it causes cognitive overhead to understand what this means.

Reply:
Are you referring to just replacing "function declaration" with "function declaration closing parenthesis" or to all the changes I made to this paragraph?

Reply:
Mostly to the "function declaration closing parenthesis".

Reply:
Done, I just pushed a new commit reverting to use "function declaration" instead of "function declaration closing parenthesis".